### DISCUSSION TITLE
Know issue: Missing Restart Policy for "cron" Container in Docker Compose Files

### DISCUSSION THREAD

**Discussion by eduardogsilva** [eduardogsilva/wireguard_webadmin#16] [Announcements]
Hello everyone,

I've recently discovered an oversight in our Docker compose files regarding the "cron" container. I forgot to specify a restart policy for this container. This oversight means that if you restart your server, the "cron" container might not automatically start on boot. Consequently, the scheduled tasks for reading and storing the latest handshake timestamp for each peer will not be executed. This container is also used for checking for new versions of `wireguard_webadmin`.

This issue affects all versions up to 0.9604.

**Action Required:** It is important to update your setup to include a restart policy for the "cron" container to ensure the correct operation. For detailed upgrade instructions, please follow the link below:

[Upgrade Instructions](https://github.com/eduardogsilva/wireguard_webadmin?tab=readme-ov-file#upgrade-instructions)

If you have any questions, please let me know!

Best regards,
Eduardo Silva

**Comment by xgusto** [comment]
I noticed it already last week. I forgot to write it to you to edit all docker-compose-***.yml
`restart: unless-stopped`
:+1:
